bootstrap/opnsense-bootstrap.sh

```diff
@@ -123,9 +123,9 @@ fi
 if [ -n "${DO_FACTORY}" ]; then
 	if [ -z "${DO_YES}" ]; then
 		echo
-		echo -n "Factory reset mode selected, are you sure? [y/N]: "
+		echo "Factory reset mode selected, are you sure? [y/N]: "
 
-		read YN
+		read -r "YN"
```

Comment:
it's hard to find command line options for builtins, not sure what -r does. Quoting literal strings is a bit over the top though :)